apps/cyberstorm-storybook/stories/components/Pagination.stories.tsx

```diff
@@ -1,11 +1,11 @@
-import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { StoryFn, Meta } from "@storybook/react";
 import { Pagination } from "@thunderstore/cyberstorm";
 import React, { useState } from "react";
 
-const meta = {
+export default {
   title: "Cyberstorm/Components/Pagination",
   component: Pagination,
-} as ComponentMeta<typeof Pagination>;
+} as Meta;
```

Comment:
Why are we dropping the `<typeof Pagination>` part here?

Reply:
As you questioned above, I did use working example as source of how it could be done and after checking some storybooks change logs made assumption that it wasn't needed there anymore. I admit that it might have been hasty decision since CSFv3 introduced new methods to use objects instead of functions to represent stories.
Should I add them back since it wouldn't be that huge work and might save time on future migration?

Reply:
Well if you're going to revert the `export` changes, might do these as well 👍🏻